lms/static/scripts/frontend_apps/components/dashboard/AssignmentActivity.tsx

```diff
@@ -44,6 +45,8 @@ export default function AssignmentActivity() {
     [students.data],
   );
 
+  useDocumentTitle(`${assignment.data?.title ?? ''}`);
```

Comment:
Maybe use something like "Untitled assignment" as the fallback and same for course activity. Otherwise we'll get a document title of "- Hypothesis" which is not very helpful.